src/daffodilDebugger/utils.ts

```diff
@@ -43,8 +43,9 @@ export const shellArgs = (port: number, isAtLeastJdk17: boolean) => {
   // Workaround: certain reflection (used by JAXB) isn't allowed by default in JDK 17:
   //   https://docs.oracle.com/en/java/javase/17/migrate/migrating-jdk-8-later-jdk-releases.html#GUID-7BB28E4D-99B3-4078-BDC4-FC24180CE82B
   const extraArgs = isAtLeastJdk17
-    ? ['-J--add-opens', '-Jjava.base/java.lang=ALL-UNNAMED']
-    : []
+  ? osCheck( ['-J"--add-opens java.base/java.lang=ALL-UNNAMED"'],
```

Comment:
I will approve this once the formatting CI stage doesn't fail. All you should need to do is run

```bash
yarn lint:fix
git commit -a -m "Fix lint issues"
```

Then squash your commits to a single commit and force push.

If you have not done you will find some instruction below.

## Squash and force push instructions
- Firstly make sure you have the upstream repo origin added to your repo

   ```bash
   git remote add upstream git@github.com:apache/daffodil-vscode.git
   git fetch --all
   ```

   - If you have done this already have this remote then only run `git fetch --all` to make sure you have the latest `main`.

- Rebase your branch off of the `main` branch.

   ```bash
   git rebase -i upstream/main
   ```

   - This will give you a least of commits such as:
      ```txt
      pick 4a7c6b4 changed assigment of extraArgs to be os specific and fix Java 17 on Windows
      pick 5d82c5a Fix lint issues
      ```
   - You will want to change every commit except the first one from `pick` to `s` or `squash`. Then after this it will give you options to update the commit message. Should look like something below:
      ```txt
      # This is a combination of 2 commits.
      # This is the 1st commit message:

      changed assigment of extraArgs to be os specific and fix Java 17 on Windows

      # This is the commit message #2:

      Fix lint issues
      ```
   - In this case you should be able to delete everything other than
      ```txt
      changed assigment of extraArgs to be os specific and fix Java 17 on Windows
      ```
- After the rebase is done you will need to force push to your branch:

   ```bash
   git push --force origin fix-java-17-parameters
   ```